src/TouristInfoCards.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const TouristInfoCards = props => {
+  return (
+    <div class='card' style={{ width: '18rem', right: '4' }}>
```

Comment:
Remember that when you're writing a React component you have to convert `class` to `className`